corehq/apps/translations/tests/test_bulk_app_translation.py

```diff
@@ -37,6 +37,89 @@
 from corehq.util.workbook_json.excel import get_workbook, WorkbookJSONReader
 
 
+EXCEL_HEADERS = (
```

Comment:
Now that this is file-level data, it would be helpful to either rename these variables or add a comment mentioning which tests they're relevant to. This file is a mess of different test apps (I have greatly contributed to it...).

Reply:
I think this is a great idea. I've merged so that GS can start testing. (I see I've *just* missed today's deploy.) I'll follow up in a separate PR.